urllib3/connection.py

```diff
@@ -204,13 +205,21 @@ class HTTPSConnection(HTTPConnection):
     default_port = port_by_scheme['https']
 
     def __init__(self, host, port=None, key_file=None, cert_file=None,
-                 strict=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT, **kw):
+                 strict=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
+                 ssl_context=None, ssl_version=None, **kw):
```

Comment:
Hrm, is it? I see it as a property on `VerifiedHTTPSConnectionPool` but it's not a param on the constructor is it? Plus it's not on any of the other pools, so I don't think it's an issue?